pytorch/inference/docker/1.5.1/py3/Dockerfile.eia

```diff
@@ -76,8 +77,7 @@ RUN conda install -c \
  && ln -s /opt/conda/bin/pip /usr/local/bin/pip3 \
  # Torchvision wheel must be installed first, so that PyTorch-EI framework is not overwritten.
  && pip install https://download.pytorch.org/whl/cpu/torchvision-0.6.1%2Bcpu-cp36-cp36m-linux_x86_64.whl \
- && pip install https://s3-us-west-2.amazonaws.com/wanchunx-pytorch-1.5.1-whl-test/pytorch-1.5.1-whl-test/torcheia-1.0.0-cp36-cp36m-manylinux1_x86_64.whl \
- # && pip install https://amazonei-pytorcheia.s3.amazonaws.com/releases/v1.0.0/torcheia-1.0.0-cp36-cp36m-manylinux1_x86_64.whl \
+ && pip install http://amazonei-pytorcheia.s3.amazonaws.com/releases/v1.0.0/torcheia-1.0.0-cp36-cp36m-manylinux1_x86_64.whl \ 
```

Comment:
Just to clarify - What does the `1.0.0` represent here?
Is this the `v1` build of Torch-EIA?
Will there be frequent updates to the version of `torcheia` used?
Is there a relation between the framework version of PyTorch used in this docker image and the `torcheia` version?

Reply:
This whl is produced through our codepipeline and it includes stock pytorch framework (`torch==1.5.1`) and eia library (`torcheia==1.0.0`). `torcheia` is decoupled from stock pytorch since 1.5.1. 
There will be a new version of `torcheia` if it gets patched or updated. We plan to release pytorch 1.6 or 1.7 next month, which means there will be `torcheia-1.1.0` later. But I don't believe the updates will be frequent. Updates depends on frequency of stock pytorch release as well as requests from customers.
We will document relationship between PyTorch framework version and torcheia version :)